lib/tasks/fixes/ap_5699.rake

```diff
@@ -0,0 +1,60 @@
+namespace :fixes do
+  desc "Fixes for AP-5699"
+  task ap_5699: :environment do |_task, args|
+    args.with_defaults(dry_run: true)
+    dry_run = args[:dry_run] != "false"
+
+    if dry_run
+      puts "Dry run mode enabled. No changes will be made."
+    else
+      # ==========
+      # LLARNI
+      # ==========
+
+      potter_derby_application_ids = %w[
+        96c3f849-e2d5-4a71-b58f-8cddbb5a4f8a
+        a6e1ad6a-0875-4b00-a0eb-21f22723f743
+        1d2376e0-29a7-4e16-96ba-dbb4bafe252d
+        7206e8ab-938f-4f95-9ce2-813989ea3568
+      ]
+
+      # Transfer applications from POTTER DERBY LTD office 0M434D to WACE MORGAN SOLICITORS office 0H036Y
+      potter_derby_application_ids.each do |appid|
+        Rake::Task["pda:laa_transfer_office"].invoke(appid, "c2d4f6df-0a06-4ade-b1c8-1bba1406a874")
+      end
+
+      # Delete the firm and offices of POTTER DERBY LTD
+      Rake::Task["pda:delete_firm_office"].invoke("210ddca4-ad8c-4401-a981-1027efe7cf60")
+
+      # ==========
+      # DSYKES
+      # ==========
+
+      # Transfer applications from CAROLE BURGHER SOLICITORS office 0M895E to BUTCHER & BARLOW LLP office 0L189A
+      Rake::Task["pda:laa_transfer_office"].invoke("6db37c7a-7350-4384-aa2d-c6a6ccfc22e9", "2a7babf2-d080-4181-b395-0244542ff54a")
+
+      # Delete the firm and offices of CAROLE BURGHER SOLICITORS
+      Rake::Task["pda:delete_firm_office"].invoke("29bbd5ca-5820-4502-b74a-76b11e8bff64")
+
+      # ==========
+      # REFERRALS@FISHCO...
+      # ==========
+
+      # Transfer provider EFERRALS@FISHCO.. from firm KEITH PARK SOLICITORS to HARPER ASHBY BOWLES LTD
+      Rake::Task["pda:provider_transfer_firm"].invoke("e4547a80-0621-44bf-a579-e7944ab5f947", "142b3c71-87b9-4105-b607-2b14ffb34e7c")
```

Comment:
Firm is transferred, but `selected_office_id` has not changed and cannot be nullable.
Discuss transferring to an office within the new firm.
Discuss making `office_codes` null